build-system/eslint-rules/forbidden-terms-config.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {readFileSync} = require('fs');
+
+/**
+ * @fileoverview
+ * Ensures that forbidden-terms.js is up-to-date by reporting listed files that
+ * do not include the allowed term.
+ */
+
+const allowlistProperty = 'allowlist';
+
+module.exports = {
+  meta: {fixable: 'code'},
+  create(context) {
+    // Some files are allowlisted for multiple terms, so caching their content
+    // greatly speeds up this rule.
+    const fileContent = {};
+
+    function readFileContent(filename) {
+      try {
+        return (
+          fileContent[filename] ||
+          (fileContent[filename] = readFileSync(filename, 'utf-8'))
+        );
+      } catch (_) {
+        return null;
+      }
+    }
+
+    function* removeFromArray(fixer, node) {
```

Comment:
I think you can just `fixer.remove(node)` and let eslint automatically fix formatting?

Reply:
[Unfortunately that won't remove the trailing comma nor the whitespace.](https://astexplorer.net/#/gist/daa3bf388358dda390877828d705dc9d/3d0cb4673e6a1c067f298c01f17d71a17bf73a7c)

I also want to remove comments on the same line, since context for those is lost.